linera-views/src/test_utils/mod.rs

```diff
@@ -8,6 +8,39 @@ use crate::batch::{
 
 use rand::{Rng, RngCore};
 use std::collections::HashSet;
+use tracing::warn;
+
+fn generate_random_alphanumeric_string(length: usize) -> String {
+    // Define the characters that are allowed in the alphanumeric string
+    let charset: &[u8] = b"0123456789abcdefghijklmnopqrstuvwxyz";
+
+    let mut rng = rand::thread_rng();
+    let alphanumeric_string: String = (0..length)
+        .map(|_| {
+            let random_index = rng.gen_range(0..charset.len());
```

Comment:
[There's a distribution in `rand` for that.](https://docs.rs/rand/latest/rand/distributions/struct.Alphanumeric.html)

Reply:
No, because we need characters in a-z and 0-9 but not A-Z.

Reply:
I am not sure how that could be done and would appreciate some details.